src/main/java/nl/tudelft/dnainator/ui/splashscreen/SplashScreen.java

```diff
@@ -38,8 +39,20 @@ public void start(Stage primaryStage) throws Exception {
 	}
 
 	@Override
+	public boolean handleErrorNotification(ErrorNotification info) {
+		new ExceptionDialog(null, info.getCause(),
+				"Could not launch the application.");
+		Platform.exit();
+		return false;
+	}
+
+	@Override
 	public void handleApplicationNotification(PreloaderNotification info) {
-		stage.hide();
+		if (info instanceof ErrorNotification) {
+			handleErrorNotification((ErrorNotification) info);
+		} else {
```

Comment:
I would prefer to have the splashscreen immediately hide when the error occurs, so you are only left with the exception.